astro/components/SiteTitle.astro

```diff
@@ -0,0 +1,76 @@
+---
+import AstrolightSiteTitle from "@astrojs/starlight/components/SiteTitle.astro";
+import type { Props } from "@astrojs/starlight/props";
+
+const menuItems = [
+  { name: "Docs", href: "/introduction" },
+  { name: "Performance", href: "/performance" },
+  { name: "Playground", href: "/playground" },
+];
+
+function pathsMatch(pathA: string, pathB: string): boolean {
+  return pathB.includes(pathA);
```

Comment:
Why not go with?

```suggestion
function pathsMatch(lhs: string, rhs: string): boolean {
  return rhs.startsWith(lhs);
```

Reply:
I don't know, but I like the names you suggest more.